src/main/java/mediathek/gui/abo/ManageAboPanel.java

```diff
@@ -92,14 +92,14 @@ private void initializeTable() {
     }
 
     private void setupToolBar() {
+      CreateNewAboAction newAboAction = new CreateNewAboAction(Daten.getInstance().getListeAbo());  
```

Comment:
What is your used operating system? Applying this change causes a NPE during runtime and use of the program on macOS. That was the original reason why it is inside the block.

Reply:
I'm using Windows 10 with Netbeans for development and Windows and Linux for execution. 
I got exceptions  therefore I moved the statement.  In the moment I can't reproduce this, I revert this change in an update.